openfe/setup/methods/rbfe_methods.py

```diff
@@ -0,0 +1,257 @@
+    rigid_water = True
+
+
+class StatePoint(BaseModel):
+    """Description of the temperature and pressure to model"""
+    class Config:
+        arbitrary_types_allowed = True
+
+    temperature = 298.15 * unit.kelvin
+    pressure = 1 * unit.bar
+
+    @validator('pressure', 'temperature')
+    def must_be_positive(cls, v):
+        if v <= 0:
+            raise ValueError("Pressure and temperature must be positive")
+        return v
+
+    @validator('pressure')
+    def is_pressure(cls, v):
+        if not v.is_compatible_with(unit.bar):
+            raise ValueError("Must be pressure value, e.g. use unit.bar")
+        return v
+
+    @validator('temperature')
+    def is_temperature(cls, v):
+        if not v.is_compatible_with(unit.kelvin):
+            raise ValueError("Must be temperature value, e.g. use unit.kelvin")
+
+
+class LambdaProtocolSettings(BaseModel):
+    """Settings for the lambda protocol
+
+    This describes a fixed number of windows
+
+    Attributes
+    ----------
+    functions : str, default 'default'
+      key of which switching functions to use for alchemical mutation
+    windows : int, default 11
+      number of lambda windows to calculate
+    sample_endstate : bool, default False
+      whether to sample the endstates ???
+    """
+    functions = 'default'
+    windows = 11
+    sample_endstates = False
+
+
+class MonteCarloBarostatSettings(BaseModel):
+    """Settings for the OpenMM Monte-Carlo Barostat
+
+    The temperature and pressure value is taken from the StatePoint variable
+
+    Attributes
+    ----------
+    frequency : unit.timestep
+      the number of timesteps between attempts of changing the pressure
+    """
+    class Config:
+        extra = 'forbid'
+
+    frequency = 50 * unit.timestep
+
+
+class MCMCLangevinSplittingDynamicsMoveSettings(BaseModel):
+    """Settings for the integrator
+
+    Attributes
+    ----------
+    timestep
+      size of timestep
+    collision_rate
+
+    n_steps
+
+    reassign_velocities
+
+    n_restart_attempts
+
+    constraint_tolerance
+    """
+    class Config:
+        arbitrary_types_allowed = True
+
+    timestep = 0.02 * unit.femtosecond
+    collision_rate = 1 / unit.picosecond
+    n_steps = 2500 * unit.timestep
+    reassign_velocities = True
+    n_restart_attempts = 20
+    constraint_tolerance = 1e-06
+
+
+class HybridTopologyFactorySettings(BaseModel):
+    use_dispersion_correction = False
+    softcore_alpha = 0.5
+    softcore_LJ_v2 = True
+    softcore_LJ_v2_alpha = 0.85
+    softcore_electrostatics = True
+    softcore_electrostatics_alpha = 0.3
+    softcore_sigma_Q = 1.0
+    interpolate_old_and_new_14s = False
+    flatten_torsions = False
+
+
+class ReporterSettings(BaseModel):
+    output_filename = 'rbfe.nc'
+    checkpoint_interval = 10
+
+
+class SimulationLengthSettings(BaseModel):
+    class Config:
+        arbitrary_types_allowed = True
+
+    minimization = 1000
+    equilibration = 5 * unit.picosecond
```

Comment:
for now yeah but long term we should have longer defaults 🤣

Reply:
I think I can take away this default to make people think about it tbh

Reply:
Can we make the default protocol a single instance rather than setting defaults on parameters that users should probably think about?